t/t2410-sdexec-memlimit.t

```diff
@@ -181,4 +179,6 @@ test_expect_success 'memory.max configuration changed' '
 	test_cmp inf.exp maxupdate.out
 '
 
+flux setattr log-stderr-level 3
+
```

Comment:
just want to double check wasn't left errantly?  Seems unnecessary right before `test_done`?

Reply:
That was on purpose.  Without it, we get a stream of module unload messages that are not helpful.